src/modules/headset/headset_openxr.c

```diff
@@ -198,13 +200,18 @@ static bool openxr_init(float supersample, float offset, uint32_t msaa) {
     for (uint32_t i = 0; i < extensionCount; i++) extensions[i].type = XR_TYPE_EXTENSION_PROPERTIES;
     xrEnumerateInstanceExtensionProperties(NULL, 32, &extensionCount, extensions);
 
-    const char* enabledExtensionNames[4];
+    const char* enabledExtensionNames[5];
     uint32_t enabledExtensionCount = 0;
 
 #ifdef __ANDROID__
     enabledExtensionNames[enabledExtensionCount++] = XR_KHR_ANDROID_CREATE_INSTANCE_EXTENSION_NAME;
 #endif
 
+    if (hasExtension(extensions, extensionCount, "XR_EXTX_overlay")) {
+      enabledExtensionNames[enabledExtensionCount++] = "XR_EXTX_overlay";
```

Comment:
It might be possible to eliminate `overlayModeActive` since it's only used in init but meh, it's sorta convenient here and may be useful outside init later.

Reply:
Ah yes that would make things simpler.